core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -229,13 +229,20 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Exter
       }
     }
 
-    if (hc.getConf.isKillOnUnhealthyClusterEnabled) {
+    startUnhealthyStateKillThread()
```

Comment:
Is this UnhealthyStateKillThread actualyl working .. we have an issue with 3.26 release and 
backend cluster can't start up - sent more details in an email